test/compilable/ddocunittest.d

```diff
@@ -1,4 +1,4 @@
-// PERMUTE_ARGS: -unittest
+// PERMUTE_ARGS: -unittest -preview=allowTypesForTemplateAlias
```

Comment:
@FeepingCreature Still need to remove this use of flag.

Reply:
fixed